### ISSUE TITLE
Rifle SFX doesn't play on Debian Stretch

### ISSUE THREAD

**Issue by d3fault** [ddnet/ddnet#1389]
The rifle SFX doesn't play when I shoot the rifle. I just updated to debian-live-9.6.0-amd64-xfce.iso -- coming from Jessie where the Rifle SFX worked fine.

I am building ddnet from source and using the system libs. I tried a bunch of older versions of ddnet, but it still didn't work.

The hammer and shotty SFX work fine. Weird.

VLC doesn't play the rifle SFX (or it "plays it silently" with no error), but in all fairness VLC doesn't play the hammer or shotty SFX either (VLC plays music_main.wv fine though). mplayer does play the rifle/hammer/shotty SFX fine.

I'm on an: Intel(R) Core(TM)2 Duo CPU     E6550  @ 2.33GHz
Audio device: Intel Corporation 82801I (ICH9 Family) HD Audio Controller (rev 02)

I have what comes with the official debian-live-9.6.0-amd64-xfce.iso , a regular /etc/apt/sources.list file, and I literally have nothing else installed except the prerequisites for building ddnet (ddrace is lyfe). Since ddnet is going to be in Debian Buster, I figured you might be interested in this bug.

**Comment by d3fault** [comment]
Fastest bug report resolution evar! I kinda half agree with the change though. Sure rifle SFX is annoying but it's also really useful to coordinate timing/etc with other players. Most players won't know [how] to turn the rifle SFX back on and will think the game is broken or something.

A better solution would be to ask the user on the first launch of ddnet whether they want to have the rifle SFX enabled/disabled. It should be a checkbox right below the dialog where you enter your name for the first time... and the rifle SFX should be opt-out, not opt-in like currently.

**Comment by d3fault** [comment]
Also snd_gun implies that it affects all the guns, not just the rifle. snd_rifle makes more sense...

**Comment by 12pm** [comment]
rifle is laser, not pistol

**Comment by def-** [comment]
Indeed, in the code it's always called gun. If we start changing names then the current setting of people gets changed, which is not something we want.
